src/pages/index.js

```diff
@@ -57,6 +58,10 @@ const StyledTabItem = styled(Link)`
   border-bottom: none;
   text-decoration: none;
 `
+const StyledTabTitle = styled(Box)`
+  color: ${WHITE};
+  ${space({ pb: 1 })}
```

Comment:
@ric9176 I'd use instead of `${space({ pb: 1 })}` directly in the style the following:
```
StyledTabTitle.defaultProps = {
    pb: 1
}
```
This way you don't have to import space from styled-components, and that's an implementation detail in the Box